@navikt/core/react/src/monthpicker/MonthPicker.tsx

```diff
@@ -0,0 +1,263 @@
+  const monthRefs = useRef(new Array<HTMLButtonElement>());
+
+  if (!fromDate) return <></>;
+  if (!toDate) return <></>;
+
+  if (isSameYear(fromDate, toDate)) {
+    const date = startOfMonth(fromDate);
+    for (let month = fromDate.getMonth(); month <= toDate.getMonth(); month++) {
+      months.push(setMonth(date, month));
+    }
+  } else {
+    const date = startOfMonth(new Date());
+    for (let month = 0; month <= 11; month++) {
+      months.push(setMonth(date, month));
+    }
+  }
+
+  const hideMonth = (month: Date) => {
+    return compareAsc(month, fromDate) === -1;
+  };
+  return (
+    <BodyShort as="div" className="navds-monthpicker__months">
+      {months.map((x: Date, y) => {
+        const currentRef = (x: any) => monthRefs.current.push(x);
+        return (
+          <button
+            ref={currentRef}
+            key={x.toDateString()}
+            data-index={y}
+            onClick={() =>
+              onSelect(setYear(startOfMonth(x), Number(selected.getFullYear())))
+            }
+            className={cl("navds-monthpicker__month", {
+              "navds-monthpicker__month--hidden": hideMonth(x),
+              "navds-monthpicker__month--current": dateIsInCurrentMonth(
+                x,
+                selected
+              ),
+              "navds-monthpicker__month--selected": dateIsSelected(x, selected),
+            })}
+            onKeyDown={(e) => {
+              if (e.key === "ArrowDown") {
+                monthRefs.current[y + 4] && monthRefs.current[y + 4].focus();
+              }
+              if (e.key === "ArrowUp") {
+                monthRefs.current[y - 4] && monthRefs.current[y - 4].focus();
+              }
+              if (e.key === "ArrowRight") {
+                e.currentTarget.nextSibling &&
+                  (e.currentTarget.nextSibling as HTMLButtonElement).focus();
+              }
+              if (e.key === "ArrowLeft") {
+                e.currentTarget.previousSibling &&
+                  (
+                    e.currentTarget.previousSibling as HTMLButtonElement
+                  ).focus();
+              }
+            }}
+          >
+            <span aria-hidden="true">
+              {format(new Date(x), "LLL", { locale })
+                .replace(".", "")
+                .substring(0, 3)}
+            </span>
+            <span className="navds-sr-only">
+              {format(new Date(x), "LLLL", { locale })}
+            </span>
+          </button>
+        );
+      })}
+    </BodyShort>
+  );
+};
+
+export const MonthPicker = forwardRef<HTMLDivElement, MonthPickerProps>(
+  (
+    {
+      children,
+      yearSelector = false,
+      fromDate = new Date(),
+      toDate = new Date("Sep 27 2032"),
+    },
+    ref
+  ) => {
+    const [selected, setSelected] = React.useState<Date>(new Date());
+
+    const isValidYearSelector =
+      yearSelector && fromDate && toDate ? true : false;
+
+    return (
+      <RootProvider
+        locale={NB}
```

Comment:
Gir det mening at `locale` alltid er `NB` her?

Reply:
Monthpicker-delen er WIP enda, så ikke helt klar for review enda